src/main/webapp/app/programming/manage/update/update-components/problem/checklist-panel/checklist-panel.component.ts

```
@@ -0,0 +1,507 @@
+                error: () => {
+                    this.alertService.error('artemisApp.programmingExercise.instructorChecklist.actions.error');
+                    this.isApplyingAction.set(false);
+                    this.actionLoadingKey.set(undefined);
+                },
+            });
+    }
+
+    private markSectionsStale(sections: ChecklistSectionType[]) {
+        this.staleSections.update((current) => {
+            const n = new Set(current);
+            sections.forEach((s) => n.add(s));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)